spec/models/mixins/supports_feature_mixin_spec.rb

```diff
@@ -263,6 +263,14 @@ def initialize(values = {})
     end
   end
 
+  let(:model) do
```

Comment:
I moved this up because it was being defined after the tests that were using it.